doc/classes/ProjectSettings.xml

```diff
@@ -191,6 +199,15 @@
 				This can also be used to erase custom project settings. To do this change the setting value to [code]null[/code].
 			</description>
 		</method>
+		<method name="unload_resource_pack">
+			<return type="bool" />
+			<argument index="0" name="pack" type="String" />
+			<description>
+				Unloads a previously loaded [code]pack[/code] from the resource filesystem. Returns [code]true[/code] on success.
```

Comment:
```suggestion
				Unloads a previously loaded [param pack] from the resource filesystem. Returns [code]true[/code] on success.
```

Reply:
This makes the pre-commit hook fail:
```
Running hook: pre-commit-make-rst
Checking for errors in the XML class reference...
ERROR: ProjectSettings.xml: Unresolved type "param pack".
1 error was found in the class reference XML. Please check the messages above.
```
Also I can't find any instance of `[param` anywhere else in documentation, so I'm not sure about this change.

Reply:
You might need to rebase your branch onto the latest master. The param syntax is a relatively new addition.

Reply:
Thanks, I forgot to do that, it's working now :+1: